Application/RSBot.NavMeshVisualizer/NavMeshRenderer.cs

```diff
@@ -0,0 +1,375 @@
+﻿using RSBot.FileSystem;
+using RSBot.NavMeshApi;
+using RSBot.NavMeshApi.Mathematics;
+using RSBot.NavMeshApi.Object;
+using RSBot.NavMeshApi.Terrain;
+
+using System.Diagnostics;
+using System.Drawing.Drawing2D;
+using System.Numerics;
+
+namespace RSBot.NavMeshVisualizer;
+
+public partial class NavMeshRenderer : UserControl
+{
+    private bool _isDragging;
+    private Point _dragPosition;
+
+    private NavMeshTransform _transform;
+    private NavMeshTransform _mouseTransform;
+    private NavMeshRaycastHit? _hit = null;
+
+    private readonly Font _font;
+    private readonly Font _smallFont;
+
+    private float _zoom = 1.0f;
+
+    private bool _drawRegionId = true;
+    private bool _drawRegionBorder = true;
+    private bool _drawTerrainGlobalEdges = true;
+    private bool _drawTerrainInternalEdges = true;
+    private bool _drawObjectGlobalEdges = true;
+    private bool _drawObjectInternalEdges = true;
+    private bool _drawObjectGround = true;
+
+    private bool _drawTerrainCellID = false;
+    private bool _drawTerrainGlobalEdgeID = false;
+    private bool _drawTerrainInternalEdgeID = false;
+    private bool _drawObjectCellID = false;
+    private bool _drawObjectGlobalEdgeID = false;
+    private bool _drawObjectInternalEdgeID = false;
+
+    public NavMeshRenderer()
+    {
+        this.InitializeComponent();
+
+        if (this.DesignMode)
+            return;
+
+        var data = new PackFileSystem(Path.Combine("D:\\Games\\Silkroad_TestIn", "Data.pk2"), "169841");
+
```

Comment:
Please use the game client path instead of the static path.

Reply:
File was obsolete by @ngoedde's anyway, not sure why he hasn't deleted it. It's gone now.